server/bundler/bin/bundler.js

```diff
@@ -106,4 +106,15 @@ webpack( webpackConfig, function( error, stats ) {
 	assets = utils.getAssets( stats.toJson() );
 
 	fs.writeFileSync( path.join( __dirname, '..', 'assets.json' ), JSON.stringify( assets, null, '\t' ) );
+
+	if ( ! process.env.WEBPACK_OUTPUT_JSON ) {
```

Comment:
I feel like I pulled this out a while ago because it didn't help anymore, once we had parallel minification working properly. @gziolo did you add this back in or did I forget to remove it from this patch?